components/ide/code/extensionhelper/main.go

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"sync"
+	"time"
+
+	"google.golang.org/grpc"
+	yaml "gopkg.in/yaml.v2"
+
+	gitpod "github.com/gitpod-io/gitpod/gitpod-protocol"
+	supervisor "github.com/gitpod-io/gitpod/supervisor/api"
+)
+
+func main() {
+	log.SetOutput(io.Discard)
+	f, err := os.OpenFile(os.TempDir()+"/.gitpod-code-helper.log", os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0644)
```

Comment:
Can we output to stdout/stderr instead? Otherwise we won't see it in GCP logs.

Reply:
Alternatively in startup.sh we could flush content of log file to stdout?

It will be though to write errors to stderr, not stdout. In case if downstream tools like GCP highlight them differently.